src/tests/helpers.js

```diff
@@ -2213,3 +2213,51 @@ export const schemaWithCustomValidationsAndConditionals = {
     },
   ],
 };
+
+export const schemaNullField = {
+  additionalProperties: false,
+  type: 'object',
+  properties: {
+    name: {
+      type: 'null',
+      title: '(Optional) Name',
+    },
+    username: {
+      type: 'string',
+      title: 'Username',
+      maxLength: 4,
+    },
+  },
+  required: ['username'],
+};
+
+export const schemaNullFieldWithInputType = {
+  additionalProperties: false,
+  type: 'object',
+  properties: {
+    name: {
+      type: 'null',
+      title: '(Optional) Name',
+      'x-jsf-presentation': {
+        inputType: 'hidden',
+      },
+    },
+    username: {
+      type: 'string',
+      title: 'Username',
+      maxLength: 4,
+      'x-jsf-presentation': {
+        inputType: 'text',
+      },
+    },
+    age: {
+      type: 'null',
+      title: 'Age',
+      maximum: 20,
+      'x-jsf-presentation': {
+        inputType: 'number',
```

Comment:
question/bug: How is this scenario possible? Having a field of `inputType: number` but with `type null`?

Reply:
I'm pretty sure this is just confusion in my head of "json type" vs "input type" 😅 will clean this up!

Reply:
Feel free to clarify [the docs](https://json-schema-form.vercel.app/?path=/docs/guides-concepts-important-concepts--docs) on what's the difference! Basically jsonType is native specs, `inputType` is opinionated UI visualization of that same field. For example a `jsonType: string` can map to multiple `inputType`: `text`, `radio`, `select`, etc..

Reply:
Will add some clarification to the docs along the same lines of what you shared. Thank you!